changelog/v1.18.0-beta8/istio-spiffe-cert-provider-address.yaml

```
@@ -0,0 +1,6 @@
+changelog:
+  - type: FIX
```

I think this would more aptly be considered `type: HELM` as we are introducing a new Helm API (https://docs.solo.io/gloo-edge/latest/reference/changelog/changelog_types/).
```suggestion
  - type: HELM
```

---

Changelog type modified.